homeassistant/components/rest/manifest.json

```diff
@@ -3,5 +3,6 @@
   "name": "RESTful",
   "documentation": "https://www.home-assistant.io/integrations/rest",
   "requirements": ["jsonpath==0.82", "xmltodict==0.12.0"],
-  "codeowners": []
+  "codeowners": [],
+  "quality_scale": "internal"
```

Comment:
I don't agree with this being internal.

Reply:
Should we remove command_line and local_ip too?

Reply:
I would say so, yeah.

Reply:
```suggestion
```